### PR TITLE
Script for coverage when autoCompleteList and autoCompleteText present in RPC

### PR DESCRIPTION
**Pull request by Aleksandriys**
@OVikhrov @aarotar please review.

### CODE REVIEW

File: test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+commonSteps:DeleteLogsFileAndPolicyTable()
+
+--[[ General Settings for configuration ]]
+Test = require('connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+function Test:AutoCompleteList_Positive()
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua]: @Aleksandriys, please rename test to have more obvious title, like "Test:Both_AutoCompleteList_and_autoCompleteText_present" or
"Test:Check_SDL_omits_AutoCompleteText"

File: test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua [hunk 2]
```diff
@@ -0,0 +1,75 @@
+---------------------------------------------------------------------------------------------
+-- Requirement summary:
+-- [SetGlobalProperties] Conditions for SDL to omit "autoCompleteText" param at request to HMI
+--
+-- Description:
+-- SDL must: tranfer SetGlobalProperties_request with <autoCompleteList> param and without (omited) <autoCompleteText> param to HMI
+-- respond with <resultCode_received_from _HMI> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua]: @Aleksandriys,
1. Description currently looks like "expected result". Please try to use description describing case: "Case when ..." 
2. please update description with particular resultCode from HMI (please specify "SUCCESS") and not <resultCode_received_from _HMI>

File: test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+--
+-- Performed steps:
+-- 1. Register Application.
+-- 2. Mobile send RPC SetGlobalProperties with <autoCompleteList> and <autoCompleteText>
+-- 3. SDL respond without <autoCompleteText>  with <resultCode_received_from _HMI> to mobile app
```
Comment (aarotar) [inline test_scripts/API/AutoCompleteList/001_ATF_SetGlobalProperties_P_AutoCompleteList_OmitedText_HAPPY_PATH.lua]: @Aleksandriys, looks like you missed "Expected result" in header. 
I guess, at least p3."SDL respond without <autoCompleteText>  with <resultCode_received_from _HMI> to mobile app" Should be in expected result.

**Comment by Aleksandriys** [comment]
@aarotar please review.

**Pull request merged**